sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5203,6 +5214,7 @@ pi_result piextUSMEnqueueMemAdvise(pi_queue Queue, const void *Ptr,
     (*Event)->ZeCommandList = ZeCommandList;
 
     ZeEvent = (*Event)->ZeEvent;
+    Queue->RefCount++;
```

Comment:
We need a comment why queue retain is needed here (and in all other places). I actually suggest to create a helper function that will do lines 5208-5217, and have a single comment there.

Reply:
Thanks. Done.